components/dashboard/src/settings/Preferences.tsx

```diff
@@ -83,35 +84,44 @@ export default function Preferences() {
                 <Tooltip content={'Early access version, still subject to testing.'} >
                     <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultIde === 'code-latest'} onClick={() => actuallySetDefaultIde('code-latest')}>
                         <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={vscode} />
+                            <img className="w-16 filter-grayscale self-center" src={vscodeInsiders} />
                         </div>
                         <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Insiders</PillLabel>
                     </SelectableCard>
                 </Tooltip>
             </div>
             <div className="mt-4 space-x-4 flex">
                 <CheckBox
-                    title="Open in Desktop IDE"
+                    title={<div>Open in Desktop IDE <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></div>}
                     desc="Choose whether you would like to open your workspace in a desktop IDE instead."
                     checked={useDesktopIde}
                     onChange={(evt) => actuallySetUseDesktopIde(evt.target.checked)} />
             </div>
             {useDesktopIde && <>
                 <div className="mt-4 space-x-4 flex">
+                    <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultDesktopIde === 'code-desktop'} onClick={() => actuallySetDefaultDesktopIde('code-desktop')}>
+                        <div className="flex justify-center mt-3">
+                            <img className="w-16 filter-grayscale self-center" src={vscode} />
+                        </div>
+                    </SelectableCard>
+                    <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultDesktopIde === 'code-desktop-insiders'} onClick={() => actuallySetDefaultDesktopIde('code-desktop-insiders')}>
+                        <div className="flex justify-center mt-3">
+                            <img className="w-16 filter-grayscale self-center" src={vscodeInsiders} />
+                        </div>
+                        <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Insiders</PillLabel>
```

Comment:
**issue**: Now the problem with re-using the same component (`PillLabel`) here is that it's creating not clear clear to the user what's the difference between these labels. Cc @loujaybee 

**suggestion**: Ideally, we should at least use a different style but do not want to push back from merging this as is. We can always iterate and improve this later, as you mentioned in https://github.com/gitpod-io/gitpod/issues/6485#issuecomment-958114262, since this is out of the scope of this PR. Let's make sure there's a follow up issue if we decide to improve the beta label later. ➿ 

| BEFORE | AFTER |
|-|-|
| <img width="720" alt="label-before" src="https://user-images.githubusercontent.com/120486/142296457-3f549d1d-371d-44c3-8a77-2b91357f3ca1.png"> | <img width="720" alt="label-after" src="https://user-images.githubusercontent.com/120486/142296461-89b942a0-be8d-4b93-a7ed-7fa00df6c536.png"> |

Reply:
Added this to this issue: https://github.com/gitpod-io/gitpod/issues/6601#issuecomment-972673757